src/plugins/vis_types/pie/public/editor/components/pie.tsx

```diff
@@ -116,6 +129,22 @@ const PieOptions = (props: PieOptionsProps) => {
           value={stateParams.isDonut}
           setValue={setValue}
         />
+        {props.showElasticChartsOptions && stateParams.isDonut && (
```

Comment:
Can you cover this functionality in the pie.test? I have other cases there, let's test that it is hidden when vislib is on or if it is a pie. 
Also can we also add a test for lens to test that the visual options menu is only visible when we have a donut partition chart and not a pie or treemap etc